libr/util/idpool.c

```diff
@@ -163,6 +163,48 @@ R_API void* r_id_storage_get(RIDStorage* storage, ut32 id) {
 	return storage->data[id];
 }
 
+// usually these shall not be used
```

Comment:
Also, providing an API and saying from its very first day "hey you should not use them" seems wrong. Moreover, if you have a next/prev API that means there is indeed some sort of order. Saying there isn't one is just wrong and not in line with this API. I would just remove this comment.